pkg/resmgr/resource-manager.go

```diff
@@ -426,6 +426,11 @@ func (m *resmgr) SetConfig(conf config.RawConfig) error {
 		return resmgrError("config from agent is empty, ignoring...")
 	}
 
+	if opt.ForceConfig != "" {
+		m.Info("ignoring config from agent because using forced configuration %s", opt.ForceConfig)
+		return fmt.Errorf("ignoring agent config because force config is enabled")
```

Comment:
nit: maybe instead ...
```suggestion
		return resmgrError("force config is enabled, ignoring agent config...")
```
so that we have resource manager-specific formatted error?

Reply:
Sure, I will send a new version.